zerver/lib/types.py

```diff
@@ -217,7 +217,7 @@ class NeverSubscribedStreamDict(TypedDict):
     subscribers: NotRequired[List[int]]
 
 
-class APIStreamDict(TypedDict):
+class DefaultStreamDict(TypedDict):
```

Comment:
This change is a temporary workaround as we do not include `stream_weekly_traffic` field in the default stream objects.

Another option could have been to use `NotRequired[Optional[int]]` type for `stream_weekly_traffic` in `APIStreamDict` itself but that would have meant that the `stream_weekly_traffic` in `APISubscriptionDict` is also not required (as it inherits `APIStreamDict`) which is not correct as we always include `stream_weekly_traffic` field in `Subscription` objects.

Reply:
I think this is fine; the right long-term plan probably involves just sending IDs for default streams in the API anyway.